app/api/apiConfig.js

```diff
@@ -476,6 +486,14 @@ export const settingsAPIs = {
             location: "Paris",
             operator: "Witness: zapata42-witness",
             contact: "telegram:Zapata_42"
+        },
+        {
+            url: "wss://citadel.li/node",
+            location: "Island"
```

Comment:
Please notice the new required format of nodes in https://github.com/bitshares/bitshares-ui/issues/1521#issuecomment-407030258

This will facilitate more sophisticated ping strategies

Reply:
ty for correction
i've updated api: https://github.com/snakebitekit/bitshares-ui/commit/2b81d14daa146fd036736c63e3c2801a879b59fc